backend/src/httpserver/httpserver/urls.py

```diff
@@ -0,0 +1,28 @@
+"""httpserver URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/4.1/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  path('', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
```

Comment:
Probably will want to delete this boilerplate comment eventually? Could also keep it around for posterity if we need to add one of these things

Reply:
Given I'm still not super comfortable with django it's good for me at least :P